admin/admin.go

```diff
@@ -135,6 +135,43 @@ func (a Admin) HeapDump(path *string, reply *string) (err error) {
 	return nil
 }
 
+// StartCpuProfile starts writing cpu profile at given path using pprof
+func (a Admin) StartCpuProfile(path *string, reply *string) (err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			pkgLogger.Error(r)
+			err = fmt.Errorf("Internal Rudder Server Error. Error: %w", r)
+		}
+	}()
+	f, err := os.OpenFile(*path, os.O_RDWR|os.O_CREATE, 0755)
+	if err != nil {
+		return err
+	}
+	pkgLogger.Info("Starting cpu profile. Writing to ", *path)
+	err = pprof.StartCPUProfile(f)
+	if err != nil {
+		pkgLogger.Info("StartCPUProfile threw error. Cpu profiling may already be running")
```

Comment:
Can we handle only the already running error?

Reply:
done